src/App.tsx

```diff
@@ -72,20 +73,37 @@ const GameLoop = () => {
   // Show feedback
   const getSocialFeedback = (feedback: ResponseItem[]): JSX.Element => {
     function constructElement(it: ResponseItem, i: number) {
-      console.log(`animate__delay-${i}`)
       switch (it.type) {
-        case 'tweet': return <Tweet fb={it} animation={`animate__delay-${i}s	animate__animated animate__backInDown`} />;
-        case 'meme': return <Meme fb={it} animation={`animate__delay-${i}s	animate__animated animate__backInDown`} />;
-        case 'article': return <News fb={it} animation={`animate__delay-${i}s	animate__animated animate__backInDown`} />;
+        case 'tweet': return <Tweet fb={it} animation={`animate__delay-${i}s	animate__animated animate__bounceIn`} />;
+        case 'meme': return <Meme fb={it} animation={`animate__delay-${i}s	animate__animated animate__bounceIn`} />;
+        case 'article': return <News fb={it} animation={`animate__delay-${i}s	animate__animated animate__bounceIn`} />;
       };
     }
-    return <div className='w-full p-2 m-2 flex flex-col justify-center items-center '>
-      {constructElement(feedback[0], 1)}
-      {constructElement(feedback[1], 2)}
-      {constructElement(feedback[2], 3)}
+    return <div className='max-w-full w-full p-2 m-2 flex flex-col justify-center items-center '>
+      {constructElement(feedback[0], 0)}
+      {constructElement(feedback[1], 1)}
+      {constructElement(feedback[2], 2)}
     </div>
   }
 
+  // Reset game
+  const reset = () => {
+    setHistory([]);
+    setEvent(firstEvent);
+    setView('start'); 
+    setEnding('');
+    setFirstTimePlaying(false);
```

Comment:
Isn't it easier to reload the page? As for the data recording I was planning to keep all playthroughs